client/www/components/dash/explorer/RecentlyDeletedAttrs.tsx

```diff
@@ -0,0 +1,146 @@
+import { Button, Dialog, SectionHeading, useDialog } from '@/components/ui';
+import config from '@/lib/config';
+import { SchemaNamespace, InstantApp, DBAttr } from '@/lib/types';
+import { useDashFetch } from '@/lib/hooks/useDashFetch';
+import useSWR from 'swr';
+import { errorToast } from '@/lib/toast';
+import { InstantReactWebDatabase } from '@instantdb/react';
+import { ClockIcon } from '@heroicons/react/24/outline';
+import { ArrowUturnLeftIcon } from '@heroicons/react/24/solid';
+import { add, formatDistanceToNow } from 'date-fns';
+import { useEffect } from 'react';
+
+const getNamesByNamespace = (attr: DBAttr): Record<string, string> => {
+  const result: Record<string, string> = {};
+  result[attr['forward-identity'][1].split('$')[1]] =
+    attr['forward-identity'][2].split('$')[1];
+
+  if (attr['reverse-identity']) {
+    result[attr['reverse-identity'][1].split('$')[1]] =
+      attr['reverse-identity'][2].split('$')[1];
+  }
+  return result;
+};
+
+export const RecentlyDeletedAttrs: React.FC<{
+  namespace: SchemaNamespace;
+  appId: string;
+  db: InstantReactWebDatabase<any>;
+}> = ({ namespace, appId, db }) => {
+  const dashResponse = useDashFetch();
+  const app = dashResponse.data?.apps?.find((a: InstantApp) => a.id === appId);
+
+  const { data, mutate } = useRecentlyDeletedAttrs(appId, app?.admin_token);
+
+  const dialog = useDialog();
+
+  const restoreAttr = async (attrId: string) => {
+    if (!db) return;
+    if (!data) return;
+    try {
+      await db._core._reactor.pushOps([['restore-attr', attrId]]);
+      mutate({
+        attrs:
+          data.attrs.filter((attr) => (attr.id === attrId ? false : true)) ??
```

Comment:
nit: I think you can just write here `attr.id !== attrId`

Reply:
🤦‍♂️